.kitchen.yml

```diff
@@ -54,19 +51,30 @@ suites:
         server_repl_password: q654321
```

Comment:
Can't comment on above lines for some reason, but do we really need these MySQL settings?